### PR TITLE
CASMTRIAGE-3505 - Upload customizations.yaml to Kubernetes after the new credentials have been encrypted

### PR DESCRIPTION
**Pull request by spillerc-hpe**
# Description

The procedure to change the VCS password uploads `customizations.yaml` to the loftsman namespace site-init secret before the new credentials have been encrypted resulting in an error next time `manifestgen` is run.

* Changed ordering of steps in the procedure so encrypted credentials are uploaded
* Fixed errors identified with directory paths relative to working directory while testing new procedure.

# Checklist Before Merging

- [X] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [X] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/MTL-1695/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by nrockershousen — APPROVED** [review]

File: operations/configuration_management/Version_Control_Service_VCS.md
Comment (mharding-hpe) [inline operations/configuration_management/Version_Control_Service_VCS.md:130]:
```suggestion
   If the above command complains that it cannot find `certs/sealed_secrets.crt`, then run the following commands to create it:
```

**Review by mharding-hpe — APPROVED** [review]

**Comment by mharding-hpe** [comment]
This PR isn't really CFS-related, so I don't think we need to code owner approval from the CFS team before merging. Regular approval process is fine, IMO.

**Pull request merged**
